02-visualization.Rmd

```diff
@@ -51,7 +55,7 @@ library(ggplot2)
 library(dplyr)
 ```
 
-```{r message=FALSE, warning=FALSE, echo=FALSE}
+```{r message=FALSE, echo=FALSE, purl=FALSE}
```

Comment:
Same thing here on choosing not to `purl`. Do you have a different target audience in mind for who might be looking at the purled R files?